akka-cluster-typed/src/main/scala/akka/cluster/typed/ClusterSingleton.scala

```diff
@@ -142,6 +146,13 @@ abstract class ClusterSingleton extends Extension {
     terminationMessage: A
   ): ActorRef[A]
 
+  /**
+   * Create, if needed, a proxy to a singleton in another data center. Will not attempt to create a Singleton Manager.
+   *
+   * @param singletonName Name of the singleton
+   * @return A proxy actor that can be used to communicate with the singleton in the cluster
+   */
+  def crossDcProxy[A](singletonName: String, settings: ClusterSingletonSettings): ActorRef[A]
```

Comment:
Hmm, shouldn't we rather have an overload with a DC param for the other one and keep the API nicely single-method:y?

Reply:
I find it odd I need to provide the behavior and shutdown message to just get a proxy. What do others think /cc @akka/akka-team